collections/_tips/developing.zh-hans.md

```diff
@@ -0,0 +1,656 @@
+    <ul>
+      <li><a href="javascript:return false">关于</a></li>
+      <li><a href="javascript:return false">新闻</a></li>
+      <li><a href="javascript:return false">门票</a></li>
+      <li><a href="javascript:return false">景点</a></li>
+    </ul>
+  </div>
+</div>
+<script>
+  !function() {
+    "use strict";
+    function e(e) {
+        "true" == e.getAttribute("aria-hidden") ? (e.setAttribute("aria-hidden", "false"), e.focus()) : e.setAttribute("aria-hidden", "true")
+    }
+    function t(e) {
+        "true" == e.getAttribute("aria-checked") ? (e.setAttribute("aria-checked", "false"), e.getElementsByTagName("span")[0].innerHTML = "Off") : (e.setAttribute("aria-checked", "true"), e.getElementsByTagName("span")[0].innerHTML = "On")
+    }
+    var a = document.getElementById("example-button");
+    a && (a.addEventListener("keydown", function(t) {
+        t.keyCode && 13 == t.keyCode && (e(document.getElementById("example-button-menu")), "true" == t.target.getAttribute("aria-expanded") ? t.target.setAttribute("aria-expanded", "false") : t.target.setAttribute("aria-expanded", "true"))
+    }), a.addEventListener("click", function(t) {
+        e(document.getElementById("example-button-menu")), "true" == t.target.getAttribute("aria-expanded") ? t.target.setAttribute("aria-expanded", "false") : t.target.setAttribute("aria-expanded", "true")
+    }));
+    var n = document.getElementById("switch");
+    n && (n.addEventListener("keydown", function(e) {
+        !e.keyCode || 32 != e.keyCode && 13 != e.keyCode || t(this)
+    }), n.addEventListener("click", function() {
+        t(this)
+    }))
+}();
+</script>
+</figure>
+<figure>
+<figcaption>{% include_cached icon.html name="code" %} 代码片段</figcaption>
+{:/}
+
+~~~javascript
+var buttonExample = document.getElementById('example-button');
+
+buttonExample.addEventListener('keydown', function(e) {
+  // 当回车键按下时切换菜单状态
+  if(e.keyCode && e.keyCode == 13) {
+    toggleMenu(document.getElementById('example-button-menu'));
+  }
+});
+
+buttonExample.addEventListener('click', function(e) {
+  // 当鼠标点击时切换菜单状态
+  toggleMenu(document.getElementById('example-button-menu'));
+});
+~~~
+
+{::nomarkdown}
+</figure>
+</div>
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="更多信息" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [键盘 2.1.1 指南](/WAI/WCAG21/quickref/#keyboard) ([理解 2.1.1 指南](/WAI/WCAG21/Understanding/keyboard))
+* **用户故事**
+  * [[Ade, 双臂功能受限的记者]](/people-use-web/user-stories/story-one/)
+  * [[Lakshmi, 盲人高级会计师]](/people-use-web/user-stories/story-three/)
+  * [[Marta, 一位盲聋的营销助理]](/people-use-web/user-stories/story-seven/)
+  * [[Elias, 患有低视力，手颤震，和轻度短期失忆症的退休人员]](/people-use-web/user-stories/story-nine/)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 尽可能避免使用CAPTCHA
+
+CAPTCHA会给用户造成很多阻碍。除了CAPTCHA，还有其他更易使用的用户输入验证方式，比如自动检测或者界面交互验证。
+如果一定需要用到CAPTCHA，确保它是容易理解的，另外提供其他可选方案，比如：
+
+* 提供至少两种解决CAPTCHA的方法
```

Comment:
@heybran "Providing more than two ways to solve the CAPTCHAs" (i.e at least 3) seems to have been translated into "Providing at least two ways to solve the CAPTCHAs"

If so, would you please update the translation?

Please note that I am using machine translation for these final checks and the tool could mislead me.

Reply:
@heybran Thanks for the edit. The meaning is now correct but it would be best to keep the original formulation "more than two". Do you think of  suitable alternative to "at least three"?